src/influx_udp.erl

```diff
@@ -110,4 +143,12 @@ code_change(_OldVsn, State, _Extra) ->
 %% ------------------------------------------------------------------
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
-
+-spec start_pool_(Name::atom(), Options::map()) -> {ok, Pid::pid()} | {error, Reason::any()}.
```

Comment:
Line 146 is too long: -spec start_pool_(Name::atom(), Options::map()) -> {ok, Pid::pid()} | {error, Reason::any()}..